README.md

```diff
@@ -1,161 +1,720 @@
+
+Only direct parents SHOULD be listed in a [`Cause`] field, as the complete history is intact [transatively][transative]. For example, in the graph above, `bafy...ambrosia` has no direct link to `bafy...agave` and `bafy...bun` has no direct link to `bafy...bean` because indirect, transative histories exists (shown in pink and orange respectively). The fact that this path crosses writers or stores is immaterial.
+
+One exception to avoiding writing redundant links in a causal history when some of those ancestors are expected to have different visibility to peers, such as when some facts are encrypted. 
+
+# 5 Prior Art
+
+This is a large amount of prior art in this area. Below are a few resources that were either direct influences, or frequently brought up as comparisons to PomoDB. They are presented here alphabetically:
+
+## 5.1 [Automerge]
+
+Automerge is a nested CRDT that is capable of representing JSON. The Automerge team have developed many optimizations and demo applications for CRDTs in [Local-First] applications.
+
+## 5.2 [Berkley Orders of Magnitude][BOOM]
+
+The BOOM project has explored Datalogs for various distributed systems use cases for well over a decade. "Disorderly programming", the [CALM][Keeping CALM] theorem, and [Dedalus] have been major inspirations for PomoDB.
+
+## 5.3 [Datasette]
+
+Datasette is a tool for exploration, publishing, and managing relational data. A common use case for it is combining data from multiple sources and exploring it in rich interfaces.
+
+## 5.4 [Datomic] & [Datahike]
+
+Datomic was the first datalog experience for a lot of people. Datahike replicates a subset of the Datomic APIs, but is also able to run in the browser.
+
+## 5.5 [Mentat]
+
+Mentat is a client-side Datalog that strongly embraces schemaless design (though domain modeling is still encouraged).
+
+> One might say that Mentat’s question is: “What if an SQLite database could store arbitrary relations, for arbitrary consumers, without them having to coordinate an up-front storage-level schema?”
+
+## 5.6 [Project Cambria]
+
+Cambria is a research project from [Ink & Switch] that attempts to solve JSON schema drift between different applications or versions. While Cambria uses lenses directly on JSON, PomoDB takes a lot of inspiration and expresses similar functionality as datalog rules.
+
+## 5.7 [RDF]
+
+The Resource Description Framework (or "RDF") is a mature data modeling and data store specification from the [W3C]. RDF has a very rich ecosystem of tools, and PomoDB draws from a lot of research based on RDF. It has been said that "PomoDB is RDF without formal ontologies and more CRDTs".
+
+## 5.8 [Soufflé]
+
+At time of writing, Soufflé is one of — if not "the" — premier extended Datalogs. It is very featureful, with numerous additions to overcome limitations in "classical" Datalog that are otherwise limitations for 
```

Comment:
> that are otherwise limitations for 

Did something get cut off here?

Reply:
LOL yeah. Not sure if I got pulled away by an interruption or if I hit the `D` key in normal mode 😅 Fixed.